fuse_constraints/src/absolute_pose_3d_stamped_constraint.cpp

```diff
@@ -64,7 +116,7 @@ void AbsolutePose3DStampedConstraint::print(std::ostream & stream) const
          << "  position variable: " << variables().at(0) << "\n"
          << "  orientation variable: " << variables().at(1) << "\n"
          << "  mean: " << mean().transpose() << "\n"
-         << "  sqrt_info: " << sqrtInformation() << "\n";
+         << "  sqrt_info: \n" << sqrtInformation() << "\n";
```

Comment:
```suggestion
         << "  sqrt_info: " << sqrtInformation() << "\n";
```
better to revert to reduce diff size.